packages/components/src/components/modal/modal.jsx

```diff
@@ -9,6 +9,9 @@ import Icon from '../icon/icon.jsx';
 import { useOnClickOutside } from '../../hooks';
 
 const ModalElement = ({
+    close_icon_color,
+    do_not_center_header_items,
```

Comment:
Suggestion for better code readability: change this prop to `center_header_items`